site/_components/external-community.njk

```diff
@@ -3,15 +3,23 @@
 
   Arguments:
     @param {string} logo: Logo path.
+    @param {string} topic: Community topic
     @param {object} cta: Object containing the following attributes.
     @param {string} url: CTA url.
     @param {string} text: CTA text.
  #}-->
-{% from 'icon.njk' import icon %} {% macro externalCommunity(logo, cta) -%}
-<figure class="external-community">
-  <img class="external-community__logo" src="{{logo}}" aria-hidden="true" alt="" />
-  <figcaption class="external-community__copy-wrapper">
-    <a href="{{cta.url}}" class="cta cta--link">{{cta.text}}</a>
-  </figcaption>
-</figure>
+{% from 'icon.njk' import icon %} {% macro externalCommunity(logo, topic, cta) -%}
+<a class="external-community" href="{{cta.url}}">
+  <figure class="external-community__liner">
+    <img class="external-community__logo" src="{{logo}}" aria-hidden="true" alt="" />
+    <figcaption class="external-community__copy-wrapper">
+      {% if topic %}
+      <strong>{{topic}}</strong>
```

Comment:
Two thoughts for both of these strong elements:
* We don't want to rely on the browser's built-in `strong` styling as we have specific font weights we want to use for bold text
* I don't believe these two instances of using `strong` are meant to signify to screen readers that the text is "…of 'strong importance'" or "…great seriousness or urgency" ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)). It looks just like a visual offset treatment. As such, a `span` with a class that increases font weight is probably better here.